src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
   const [releaseDate, setReleasedate] = useState(
     editableMode ? formData?.release_date : ``
   );
+  return (
+    <Formik
+      initialValues={{
```

Comment:
You need to pass here `formData` in case of editing video and some `initialFormData` in case of adding video.
`initialValues={...(editableMode ? formData : initialFormData)}`